test/rules/structured-imports/ancestorCases.ts.lint

```diff
@@ -0,0 +1,11 @@
+import * as relative from "./relativeImport1";
```

Comment:
Needs more granular test files that verify one aspect of the rule:
`no-absolute-before-external`
`no-parent-before-absolute`
`no-sibling-before-parent`
`valid-order`